test/cctest/test_util.cc

```diff
@@ -298,3 +299,17 @@ TEST(UtilTest, SPrintF) {
   const std::string with_zero = std::string("a") + '\0' + 'b';
   EXPECT_EQ(SPrintF("%s", with_zero), with_zero);
 }
+
+TEST(UtilTest, SimdutfEndiannessDoesNotMeanEndianness) {
+  // In simdutf, "LE" does *not* refer to Little Endian, it refers
```

Comment:
CC @lemire

Reply:
This is a fair point.

Reply:
Under a big-endian IBM S390 image (using `qemu`), the transcoding works fine with simdutf. I am working on adding BE unit tests downstream.

Reply:
Yeah, the point here is less about actual compatibility with big endian machines – the code looks fine for that, and this test here also passes in CI – it’s more about the functions’ naming which is confusing (in particular, it’s odd for a function that writes to an `char16_t` array to specify endianness at all, the fact that it’s `char16_t*` and not `char*` usually already implies that host endianness is being used).

Reply:
@addaleax This will be improved in future releases.